src/commands/compose/compose.ts

```diff
@@ -14,6 +14,10 @@ import { selectComposeCommand } from '../selectCommandTemplate';
 import { getComposeServiceList } from './getComposeServiceList';
 
 async function compose(context: IActionContext, commands: ('up' | 'down' | 'upSubset')[], message: string, dockerComposeFileUri?: vscode.Uri, selectedComposeFileUris?: vscode.Uri[]): Promise<void> {
+    if (!vscode.workspace.isTrusted) {
+        throw new UserCancelledError('enforceTrust');
```

Comment:
Here and elsewhere, we could throw a descriptive error (with error reporting button disabled) instead. Thoughts?

Note, the only way to even get into this code path in an untrusted workspace--because the commands are not shown, via `when` clauses--is keyboard shortcuts directly to the command, or programmatically.

Reply:
I would do that. After spending some weeks cleaning up issues coming from users hitting the "Report issue" button without providing any context, I am all for using issue reporting in a "judicious" way ☺️

Reply:
As I have it written now, it will just...do nothing. `UserCancelledError` doesn't show anything at all. Since it shouldn't even be possible to execute these commands in the first place, I felt it was OK.

I don't feel strongly one way or the other. Do you have a preference?

Reply:
If it does not show the UI for submitting an issue, then I think it is good as-is